### DISCUSSION TITLE
EME Frontend Submit: How set property values

### DISCUSSION THREAD

**Discussion by teamtacho** [liedekef/events-made-easy#504] [General]
Hi Community, Hi Franky
I want to use the Events Made Easy Frontend Submit plugin form to set a properrty value for the event.
e.g. binary value <?php EMEFS::property('allianzhm','binary'); ?>
I see that the value is stored into the database property field as "all_day":1,","allianzhm":"1"
But this value is not used in the event. like the "all_day".
I do not figured out where the custom property fields are stored in the database.
So, please can you guide me, who I can implement custom fields and there values into the form in order to set them within the event.

Best Regards Michael

**Comment by liedekef** [comment]
This is not the forum for the frontend submit form, please use the wordpress support forum in the future.
Next to that, from the readme:
```
= How about custom event or location fields? =
If you have defined custom fields of type Events or Locations in EME, they are part of the event or location definition.
In that case, you can also ask in the frontend for that info, by using EMEFS::custom_field('xxx'). 'xxx' can be the id or the fieldname.
Example (under the assumption 'my_price' is a custom event or location field): <?php EMEFS::custom_field('my_price'); ?>
Example (under the assumption '15' is a custom event or location field): <?php EMEFS::custom_field(15); ?>
Example (under the assumption '15' is a custom event or location field): <?php EMEFS::custom_field('15'); ?>

```

**Comment by teamtacho** [comment]
Thank you, Franky.
Sorry for posting that in the wrong forum.
